--- gdk-pixbuf/io-gdip-utils.c.orig	2009-08-29 04:07:53.000000000 +0200
+++ gdk-pixbuf/io-gdip-utils.c	2010-10-06 23:54:22.000000000 +0200
@@ -353,7 +353,6 @@
   if (!hg)
     return NULL;
 
-  IStream_SetSize (stream, *(ULARGE_INTEGER *)&size64);
   hr = CreateStreamOnHGlobal (hg, FALSE, (LPSTREAM *)&stream);
 
   if (!SUCCEEDED (hr)) {
@@ -362,6 +361,7 @@
     return NULL;
   }
   
+  IStream_SetSize (stream, *(ULARGE_INTEGER *)&size64);
   status = GdipCreateBitmapFromStream (stream, &bitmap);
 
   if (Ok != status)
