--- win32/Makefile.gcc	2010-04-19 03:35:55.000000000 +0200
+++ win32/Makefile.gcc.new	2010-08-28 18:32:51.000000000 +0200
@@ -23,14 +23,14 @@
 # If the platform is *not* MinGW (e.g. it is Cygwin or UWIN),
 # the DLL name should be changed from "zlib1.dll".
 
-STATICLIB = libz.a
+STATICLIB = libzdll.a
 SHAREDLIB = zlib1.dll
-IMPLIB    = libzdll.a
+IMPLIB    = libz.a
 
 #
 # Set to 1 if shared object needs to be installed
 #
-SHARED_MODE=0
+SHARED_MODE=1
 
 #LOC = -DASMV
 #LOC = -DDEBUG -g
@@ -59,9 +59,6 @@
 INSTALL = $(CP)
 RM = rm -f
 
-prefix = /usr/local
-exec_prefix = $(prefix)
-
 OBJS = adler32.o compress.o crc32.o deflate.o gzclose.o gzlib.o gzread.o \
        gzwrite.o infback.o inffast.o inflate.o inftrees.o trees.o uncompr.o zutil.o
 OBJA =
